docs/website/data/integrations.yaml

```diff
@@ -1,6 +1,32 @@
 # This file is used to generate the OPA ecosystem page: https://www.openpolicyagent.org/docs/latest/ecosystem/
 # For instructions on how to add an entry, see https://github.com/open-policy-agent/opa/tree/main/docs#integration-index
 integrations:
+
+  torque:
+     title: Torque
```

Comment:
I'm not sure if I just didn't see it before or if something changed, but it looks like 3 spaces are used for indentation rather than 2.

Reply:
Besides that, this looks good now.